packages/webapp/src/components/Table/Cell/CellTypes/RightChevronLink.tsx

```diff
@@ -14,18 +14,20 @@
  */
 import { IconButton } from '@mui/material';
 import { BsChevronRight } from 'react-icons/bs';
-import history from '../../../../history';
 import styles from '../styles.module.scss';
+import { Link } from 'react-router-dom';
 
 export type RightChevronLinkProps = {
   path: string;
 };
 
 const RightChevronLink = ({ path }: RightChevronLinkProps) => {
   return (
-    <IconButton color="info" className={styles.rightChevronLink} onClick={() => history.push(path)}>
-      <BsChevronRight />
-    </IconButton>
+    <Link to={{ pathname: path }}>
```

Comment:
ChevronLink cell type has been updated to use react router Link which has some nice functionality like the hover that shows your target destination in the bottom left.

Reply:
Uhm not toooo hard...the onClick behaviour has to be reformatted to the Link way.  I originally copied the other list item type and specialized it so I would love to see the two list item types merged into one generic or at least match for that too.